Packages/com.unity.inputsystem/InputSystem/Editor/AssetImporter/InputActionImporter.cs

```diff
@@ -212,12 +210,51 @@ public override void OnImportAsset(AssetImportContext ctx)
             InputActionEditorWindow.RefreshAllOnAssetReimport();
         }
 
+        internal static IEnumerable<InputActionReference> LoadInputActionReferencesFromAsset(InputActionAsset asset)
+        {
+            //Get all InputActionReferences are stored at the same asset path as InputActionAsset
+            return AssetDatabase.LoadAllAssetsAtPath(AssetDatabase.GetAssetPath(asset)).Where(
+                o => o is InputActionReference && o.name != "InputManager").Cast<InputActionReference>();
```

Comment:
Do we need this part?
 && o.name != "InputManager

InputManager data shouldn't be an InputActionReference should it ?

Reply:
You're right, in theory it shouldn't. 
But when removing an input action reference from the asset, it seems to show InputManage in the picker window. This was a just a quick fix. Something to improve, depending on the time we have to understand what's happening in InputManager.asset.